app/coins/btc.js

```diff
@@ -65,30 +65,35 @@ module.exports = {
 	logoUrlsByNetwork:{
 		"main":"./img/network-mainnet/logo.svg",
 		"test":"./img/network-testnet/logo.svg",
+		"testnet4":"./img/network-testnet/logo.svg",
 		"regtest":"./img/network-regtest/logo.svg",
 		"signet":"./img/network-signet/logo.svg"
 	},
 	coinIconUrlsByNetwork:{
 		"main":"./img/network-mainnet/coin-icon.svg",
 		"test":"./img/network-testnet/coin-icon.svg",
+		"testnet4":"./img/network-testnet/coin-icon.svg",
 		"signet":"./img/network-signet/coin-icon.svg",
 		"regtest":"./img/network-regtest/coin-icon.svg"
 	},
 	coinColorsByNetwork: {
 		"main": "#F7931A",
-		"test": "#1daf00",
+		"test": "#8f9a22",
+		"testnet4": "#1daf00",
```

Comment:
I wonder why do you change the testnet3 coin color at this point? I think both testnet3 & 4 could use the same color green.

And also I wonder if this also changes the color of the favicon, which follows coin color as well in order to visually distinguish browser tabs.

Reply:
i picked a color between mainnet and testnet to make it visually distinct, but now that i look closer it doesnt look like these colors are even used. the favicons are hardcoded in their respective `.svg`s, and the tab color is hardcoded here: https://github.com/janoside/btc-rpc-explorer/blob/153aa0ae7f902562bd1cfa2bbcea4c00026e5aef/views/layout.pug#L49C45-L50C1